index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <header id="header">
+      <img
+        src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"
+        id="header-img"
+      />
+
+      <nav id="nav-bar">
+        <ul>
+          <li><a href="#Features" class="nav-link">Features</a></li>
+          <li><a href="#video" class="nav-link">How it Works</a></li>
+          <li><a href="#Pricing" class="nav-link">Pricing</a></li>
+        </ul>
```

Comment:
Please center items vertically

Reply:
- [ ] kindly add clear reason for the change

Reply:
Please center items vertically to have a more organized navbar